src/mmg3d/mmg3d1.c

```diff
@@ -807,34 +821,41 @@ MMG5_int MMG5_movtet(MMG5_pMesh mesh,MMG5_pSol met, MMG3D_pPROctree PROctree,
                 if( !ier )  continue;
                 else if ( ier>0 )
                   ier = MMG5_movbdynompt(mesh,met,PROctree,listv,ilistv,lists,ilists,improveVolSurf);
-                else
+                else{
+                  MMG5_show_tet_location(mesh, pt, k);
                   return -1;
+                }
               }
             }
             else if ( ppt->tag & MG_GEO ) {
               ier=MMG5_boulesurfvolp(mesh,k,i0,i,listv,&ilistv,lists,&ilists,0);
               if ( !ier )  continue;
               else if ( ier>0 )
                 ier = MMG5_movbdyridpt(mesh,met,PROctree,listv,ilistv,lists,ilists,improveVolSurf);
-              else
+              else{
+                MMG5_show_tet_location(mesh, pt, k);
                 return -1;
+              }
             }
             else if ( ppt->tag & MG_REF ) {
               ier=MMG5_boulesurfvolp(mesh,k,i0,i,listv,&ilistv,lists,&ilists,0);
               if ( !ier )
                 continue;
               else if ( ier>0 )
                 ier = MMG5_movbdyrefpt(mesh,met,PROctree,listv,ilistv,lists,ilists,improveVolSurf);
-              else
+              else{
+                MMG5_show_tet_location(mesh, pt, k);
```

Comment:
Please, keep this in debug mode only (`-d` command line option, `mesh->info.ddebug` variable inside the code) : 
```
if ( mesh->info.ddebug ) {
   MMG5_show_tet_location(mesh, pt, k);
}
```